frameworks.md

```diff
@@ -8,8 +8,8 @@ we can have empty tests.
 
 Tests are grouped in `suites`, which can also be `nested`. Tests can be placed also outside of a suite, we call them *global tests*.
 
-Internally Mocha wraps everything in a suite, we call it *global suite*, so the global tests will become the tests of the aformentioned 
-suite, as also all other top level suites will become its direct child suites, implicitly all other suites will become its more deeper child suites, in a recursive structure. This is great for our CRI, because it is really what we need 
+Internally Mocha wraps everything in a suite, we call it *global suite*, so the global tests will become the tests of the aformentioned
```

Comment:
No need for line breaks in prose like this.

PS: While http://contribute.jquery.org/style-guide/prose/ doesn't mention line breaks, it might have some other useful advice